src/libcollections/dlist.rs

```diff
@@ -402,6 +410,51 @@ impl<T> DList<T> {
     pub fn pop_back(&mut self) -> Option<T> {
         self.pop_back_node().map(|box Node{value, ..}| value)
     }
+
+    /// Splits the list into two at the given index. Returns everything after the given index,
+    /// including the index.
+    ///
+    /// This operation should compute in O(n) time.
```

Comment:
fwiw I'm happy to leave it like this, to be handled by a more comprehensive collection doc conventions pass (which I'm sort-of working on).